USAGE.md

```diff
@@ -0,0 +1,561 @@
+will open a dialog to save a layer configuration file to your local computer. This contains the configuration for the
+techniques that have been customized
+(commented, colored, assigned a score, or disabled) as well as the scoring gradient setup, filter selection, layer name,
+layer description, view configuration.
+
+Saved layer configuration files can be opened in the ATT&CK Navigator to restore a layer you've worked on previously. To
+do so, open a new tab and open the "Open Existing Layer" panel. Then click "Upload from local", and select your saved
+configuration file. Doing so will restore your saved layer to the ATT&CK Navigator. This interface also has a "load from
+URL"input allowing you to open a layer json from a remote source.
+
+## Creating Layers from Other Layers
+
+Layers can be created which inherit properties from other layers. Several fields exist which can be used to choose which
+layers to inherit properties from:
+
+- <b>Domain:</b> choose the domain and version for the new layer. Layers can only inherit properties from other layers
+  of the same domain and version.
+
+- <b>Score Expression:</b> Technique <a href="#scoring-techniques">scores</a> in the created
+  layer are initialized to the result of this expression. This field should take the form of an equation or constant,
+  using variables for layers.Layer variables are shown in yellow on tabs when the "create layer from other layers" panel
+  is opened:  
+  <img src="nav-app/src/assets/image_scoreVariableExample.png" height="100px">
+
+  Each technique's score is created independently using the score expression. For example, with a score expression
+  of <code>a+b</code>, some technique <i>t</i> in the output layer would be the sum of <i>t<sub>a</sub></i> and <i>
+  t<sub> b</sub></i>.
+
+  Expressions can also be comparative, for example <code>a>b</code> will create a layer with a score of 1
+  whereever <code><a>b</code> and 0 whereever <code>a<=b</code>. Such boolean expressions can be extended using <code>
+  and</code>, <code>or</code>, <code>xor</code> and <code>not</code>. You can also use ternary expressions such
+  as <code>a > b ? 25 : 10</code>. See <a href="http://mathjs.org/docs/expressions/syntax.html#operators">this page</a>
+  for a full list of operators.
+
+  Some example score expressions:
+    - <code>(a+b)/2</code> (average two layers)
+    - <code>a+b+c</code>
+    - <code>a*(b+c)</code>
+    - <code>100-a</code> (reverse the scoring of <code>a</code>, assuming <code>a</code> is on a 0-100 scale)
+    - <code>(a>b) and (a>=75)</code>
+
+- <b>Gradient:</b> choose which layer to inherit <a href="#scoring-gradient">scoring
+  gradient</a> from.
+- <b>Coloring:</b> choose which layer to inherit <a href="#assigning-manual-colors">manually
+  assigned technique colors</a>
+- <b>Comments:</b> choose which layer to inherit <a href="#adding-comments-to-techniques">
```

Comment:
This link for and others like it seem to be broken in the GH view, the actual fragment is `#-adding-comments-to-techniques`. Probably because there is an image in the header ([related comment](https://github.com/mitre-attack/attack-navigator/pull/316/files/cd270a767ce3b0839a80c5fec218ff32fca47d08..9ff30b8cc50e792b6f7f9aabac34a220fe6fb4b5#r665077245))?